plugins/metagen.md

```diff
@@ -0,0 +1,150 @@
+<title>Eleventy Plugin Meta Generator</title>
+<meta name="author" content="Tanner Dolby">
+<meta name="title" content="Eleventy Plugin Meta Generator">
+<meta name="description" content="An eleventy shortcode for generating meta tags.">
+<meta name="generator" content="eleventy">
+<!-- Open Graph -->
+<meta property="og:type" content="website">
+<meta property="og:url" content="https://tannerdolby.com">
+<meta property="og:locale" content="en_US">
+<meta property="og:title" content="Eleventy Plugin Meta Generator">
+<meta property="og:description" content="An eleventy shortcode for generating meta tags.">
+<meta property="og:image" content="https://tannerdolby.com/images/arch-spiral-large.jpg">
+<meta property="og:image:alt" content="Archimedean Spiral">
+<!-- Twitter -->
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@tannerdolby">
+<meta name="twitter:creator" content="@tannerdolby">
+<meta name="twitter:url" content="https://tannerdolby.com">
+<meta name="twitter:title" content="Eleventy Plugin Meta Generator">
+<meta name="twitter:description" content="An eleventy shortcode for generating meta tags.">
+<meta name="twitter:image" content="https://tannerdolby.com/images/arch-spiral-large.jpg">
+<meta name="twitter:image:alt" content="Archimedean Spiral">
+<link rel="canonical" href="https://tannerdolby.com">
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="design.css">
+<style>h1 { color: #f06; }</style>
+<script src="foo.js"></script>
+<script src="bar.js" async></script>
+<script>console.log('hello, world');</script>
+```
+
+## Custom Usage
+For baseline social share functionality, providing arguments shown in the example usage above is recommended. If you want to add more tags not listed in the example, have a look at the plugin docs. You might only need a few `<meta>` tags instead of the whole set, simply provide the arguments you need and the ones not included won't generate `<meta>` tags.
+
+Template variables can be used as arguments in Nunjucks and Liquid without the curly braces or quotes. See the [metagen docs](https://metagendocs.netlify.app/) for more details on plugin usage.
+
+## Use Your Template Data
+To make your metadata dynamic, you can use template data as arguments to the shortcode without quotes or braces. The following example is within a Nunjucks (.njk) file:
+
+```nunjucks
+---
+title: Some title
+desc: Some description
+metadata:
+  title: Some other title
+  desc: Some other description
+url: https://tannerdolby.com
+image: https://tannerdolby.com/images/arch-spiral-large.jpg
+alt: Archimedean spiral
+type: summary_large_image 
+twitter: tannerdolby
+name: Tanner Dolby
+---
+{% metagen
+    title=title or metadata.title,
+    desc=desc or metadata.desc,
+    url=url + page.url,
+    img=image,
+    img_alt=alt,
+    twitter_card_type=type,
+    twitter_handle=twitter,
+    name=name
+%}
+```
```

Comment:
Same here, I think this code example should be wrapped in `{% raw %}..{% endraw %}` to avoid 11ty trying to utilize the template logic inside the snippets.